Gemfile.lock

```diff
@@ -463,4 +463,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.3.18
```

Comment:
The bundler seems to have downgraded. (Picked up your local version?)
Do can do `gem install bundler:2.3.18` so that it stays on this version as in pipelines.

Reply:
Done.